PiwikTracker/PiwikTracker.m

```diff
@@ -1379,13 +1385,16 @@ - (NSString*)platformName  {
     if ([platform isEqualToString:@"iPhone6,2"])    return @"iPhone 5s (Global)";
     if ([platform isEqualToString:@"iPhone7,1"])    return @"iPhone 6+";
     if ([platform isEqualToString:@"iPhone7,2"])    return @"iPhone 6";
+    if ([platform isEqualToString:@"iPhone8,1"])    return @"iPhone 6s";
+    if ([platform isEqualToString:@"iPhone8,2"])    return @"iPhone 6s+";
```

Comment:
What about iPhone 7 models? Here is a good overview: http://www.everyi.com/by-identifier/ipod-iphone-ipad-specs-by-model-identifier.html